awx/ui_next/src/screens/User/UserTokenDetail/UserTokenDetail.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useCallback } from 'react';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Button } from '@patternfly/react-core';
+
+import AlertModal from '../../../components/AlertModal';
+import { CardBody, CardActionsRow } from '../../../components/Card';
+import DeleteButton from '../../../components/DeleteButton';
+import { DetailList, Detail } from '../../../components/DetailList';
+import ErrorDetail from '../../../components/ErrorDetail';
+import { formatDateString } from '../../../util/dates';
+import { TokensAPI } from '../../../api';
+import useRequest, { useDismissableError } from '../../../util/useRequest';
+
+function UserTokenDetail({ token, canEditOrDelete, i18n }) {
+  const { scope, description, created, modified, summary_fields } = token;
+  const history = useHistory();
+  const { id, tokenId } = useParams();
+  const { request: deleteTeam, isLoading, error: deleteError } = useRequest(
+    useCallback(async () => {
+      await TokensAPI.destroy(tokenId);
+      history.push(`/users/${id}/tokens`);
+    }, [tokenId, id, history])
+  );
+
+  const { error, dismissError } = useDismissableError(deleteError);
+
+  return (
+    <CardBody>
+      <DetailList>
+        <Detail
+          label={i18n._(t`Application`)}
+          value={summary_fields?.application?.name}
+          dataCy="team-detail-name"
+        />
+        <Detail label={i18n._(t`Description`)} value={description} />
+        <Detail label={i18n._(t`Scope`)} value={scope} />
+        <Detail label={i18n._(t`Created`)} value={formatDateString(created)} />
```

Comment:
Hey @AlexSCorey, is there any particular reason to use `Detail` over `UserDateDetail`?

<img width="1443" alt="image" src="https://user-images.githubusercontent.com/9053044/89220234-3261ad80-d59f-11ea-8bdc-9d7736ed2ca8.png">

Reply:
no, I should have used that.  Thanks for catching that.